ui/raidboss/data/04-sb/dungeon/ala_mhigo.js

```diff
@@ -134,112 +134,201 @@
-        'Integrated Aetheromodulator': 'Linearbeschleuniger',
-        'Magitek Disruptor': 'Magitek-Disruptor',
-        'Mindjack': 'Gehirnwäsche',
-        'Magitek Ray': 'Magitek-Laser',
-        'Demimagicks': 'Demimagie',
-
-        'Art Of The Swell': 'Kunst des Windes',
-        'Art Of The Storm': 'Kunst des Sturmes',
-        'Art Of the Sword': 'Kunst des Schwertes',
+        'Storm, Swell, Sword': 'Wind, Sturm, Schwert',
+        'Storm?/Swell?': 'Wind?/Sturm?',
+        'Storm?/Swell?/Sword?': 'Wind?/Sturm?/Schwert?',
+        'Tail Laser': 'Schweiflaser',
+        'Target Search': 'Zielsucher',
+        'Unknown_206E': 'Unknown_206E', // FIXME
         'Unmoving Troika': 'Unbewegte Troika',
         'Vein Splitter': 'Erdader-Spalter',
-        'Lightless Spark': 'Lichtloser Funke',
-        'Concentrativity': 'Konzentriertheit',
-        'Storm, Swell, Sword': 'Wind, Sturm, Schwert',
       },
     },
     {
       'locale': 'fr',
       'replaceSync': {
+        'Aulus Mal Asina': 'Aulus mal Asina',
         'Magitek Scorpion': 'scorpion magitek',
-        'Aulus Mal Asina': 'Aulus Mal Asina',
-        'Zenos Yae Galvus': 'Zenos Yae Galvus',
-
+        'Prototype Bit': 'drone prototype',
         'Rhalgr\'s Gate': 'Porte de Rhalgr',
         'The Chamber of Knowledge': 'Chambre du Savoir',
         'The Hall of the Griffin': 'Salle du Griffon',
+        'The Storm': 'Spirale du chaos',
+        'Zenos Yae Galvus': 'Zenos yae Galvus',
+        'is no longer sealed': 'is no longer sealed', // FIXME
       },
       'replaceText': {
-        'Electromagnetic Field': 'Champ électromagnétique',
-        'Target Search': 'Recherche de cible',
-        'Lock On': 'Verrouillage',
-        'Tail Laser': 'Laser caudal',
-
-        'Mana Burst': 'Explosion de mana',
-        'Order To Charge': 'Ordre d\'attaquer',
-        'Order To Fire': 'Ordre de tirer',
         'Aetherochemical Grenado': 'Grenade magitek',
+        'Art of the Storm': 'Art de l\'orage',
+        'Art of the Swell': 'Art de la tempête',
+        'Art of the Sword': 'Art de l\'épée',
+        'Concentrativity': 'Kenki concentré',
+        'Demimagicks': 'Demimagie',
+        'Electromagnetic Field': 'Champ électromagnétique',
         'Integrated Aetheromodulator': 'Rayon accélérateur',
+        'Lightless Spark': 'Étincelle sans lueur',
+        'Lock On': 'Verrouillage',
         'Magitek Disruptor': 'Disrupteur magitek',
-        'Mindjack': 'Détournement cérébral',
         'Magitek Ray': 'Rayon magitek',
-        'Demimagicks': 'Demimagie',
-
-        'Art Of The Swell': 'Art de la tempête',
-        'Art Of The Storm': 'Art de l\'orage',
-        'Art Of the Sword': 'Art de l\'épée',
+        'Mana Burst': 'Explosion de mana',
+        'Mindjack': 'Contrainte mentale',
+        'Order To Charge': 'Ordre d\'attaquer',
+        'Order To Fire': 'Ordre d\'attaquer',
+        'Storm, Swell, Sword': 'Tempête, orage, épée',
+        'Storm?/Swell?': 'Tempête?/orage?',
```

Comment:
Aren't these question marks treated as regex question marks?

Reply:
If they are, it's news to me. I was under the impression when I wrote this in English that the replacements are plain strings.

Reply:
Belatedly, the replacements on the right hand side are plain strings, but the left hand side keys are strings that are interpreted as regular expressions.